pkgs/by-name/ho/horcrux/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "horcrux";
+  version = "unstable-2023-09-19";
+
+  src = fetchFromGitHub {
+    owner = "jesseduffield";
+    repo = "horcrux";
+    rev = "5e848abcca49a7ad359f5a24ef4ca7e0eda80889";
+    sha256 = "sha256-YOu3qJadfyA6MKW8OFLr0pFjGMOgCGie2f8VbG79IY0=";
+  };
+
+  vendorHash = null;
+
+  meta = with lib; {
+    homepage = "https://github.com/jesseduffield/horcrux";
+    description = "Split your file into encrypted fragments so that you don't need to remember a passcode";
+    maintainers = with maintainers; [ mh ];
+    license = licenses.mit;
+    mainProgram = "horcrux";
+  };
```

```suggestion
  meta = {
    homepage = "https://github.com/jesseduffield/horcrux";
    description = "Split your file into encrypted fragments so that you don't need to remember a passcode";
    maintainers = with lib.maintainers; [ mh ];
    license = lib.licenses.mit;
    mainProgram = "horcrux";
  };
```

https://github.com/NixOS/nixpkgs/issues/208242

---

I've seen a preference for sorting meta attributes alphabetically, would you agree, @AndersonTorres?

---

Yep!